<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HabitsWindow</class>
 <widget class="QDialog" name="HabitsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="backToMenuButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(170, 170, 127);
gridline-color: rgb(108, 108, 108);
background-color: rgb(230, 230, 230);
selection-background-color: rgb(205, 225, 255);</string>
   </property>
   <property name="text">
    <string>Back to menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 500 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>HABITS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteHabitButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(170, 170, 127);
gridline-color: rgb(108, 108, 108);
background-color: rgb(230, 230, 230);
selection-background-color: rgb(205, 225, 255);</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineHabitEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 235, 235)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>650</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(170, 170, 127);
gridline-color: rgb(108, 108, 108);
background-color: rgb(230, 230, 230);
selection-background-color: rgb(205, 225, 255);</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Create new habit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addHabitButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(170, 170, 127);
gridline-color: rgb(108, 108, 108);
background-color: rgb(230, 230, 230);
selection-background-color: rgb(205, 225, 255);</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>911</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="habitLayout"/>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>620</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Healthy habits - healthy life</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
